src/services/PodDatabase.ts

```diff
@@ -96,11 +97,16 @@ export default class PodDatabase {
   }
 
   public async addPod (params: AddPodParams) {
+    const existing = this.pods.get(params.id)
+    const uptime = existing && existing.totalUptime || 0 + Number(params.duration)
```

Comment:
Can you add parentheses to this line? Right now it will return `totalUptime` if its defined and `0 + Number(params.duration)` if not. Not sure if that's the intention, though.